### PR TITLE
Add releasenote for pr 36277

### PR DESCRIPTION
**Pull request by lcc3108**
**Please provide a description of this PR:**
Add releasenote for pr #36277
request by https://github.com/istio/istio/pull/36721#issuecomment-1007725310

### CODE REVIEW

File: releasenotes/notes/36277.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+kind: bug-fix
+area: telemetry
+issue:
+  - 35750
+  - 36166
+  - 36521
+
+releaseNotes:
+  - |
+    **Fixed** an issue when setting distribution tracing to telemetry api, send a malfomed host header.
```
Comment (bianpengyuan) [inline releasenotes/notes/36277.yaml:11]:
```suggestion
    **Fixed** an issue where enabling tracing with telemetry API will cause malformed host header being used at the trace report request.
```

Comment (lcc3108) [reply releasenotes/notes/36277.yaml:11]: I used a translator because I was not very good at English, but it looks better.
thank you

**Review by bianpengyuan — APPROVED** [review]

**Comment by ericvn** [comment]
Since the original PR was also cherry-picked into 1.11 and 1.12, the release-note should as well.

**Pull request merged**